hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/services/TestAbfsClient.java

```diff
@@ -103,7 +102,7 @@ public TestAbfsClient(){
   }
 
   private String getUserAgentString(AbfsConfiguration config,
-      boolean includeSSLProvider) throws MalformedURLException {
+      boolean includeSSLProvider) throws IOException {
```

Comment:
Why is this change required ?

Reply:
AbfsClient constructor could throw IOException, wiuth the new encoding and hashing logic included